packages/angular/src/migrations/update-20-2-0/migrate-mf-imports-to-new-package.ts

```diff
@@ -17,6 +17,9 @@ export default async function migrateMfImportsToNewPackage(tree: Tree) {
       (dep) => dep.target === 'npm:@nx/webpack'
     );
     if (usesNxWebpack) {
+      if (!graph.nodes[project]) {
+        continue;
+      }
```

Comment:
nit: we could do this check at the start of the loop to avoid searching for deps

Reply:
v.good point